linux57-tkg/README.md

```diff
@@ -32,8 +32,38 @@ You can enable support for it at the beginning of the PKGBUILD file. Make sure t
 - **optional** ZFS fpu symbols
 - **optional** Fsync support (proton)
 
+
+## Install procedure
+
+### Arch & derivatives
 ```
 git clone https://github.com/Frogging-Family/linux-tkg.git
 cd linux-tkg/linux57-tkg
+# Edit customization.cfg file 
```

Comment:
Is the header formatting intentional here? It feels out of place.

Reply:
I am not sure I understood, you mean the space after the "#" ?

Reply:
Oh I see, you wanted to explicitly comment it out. Okay.